palette/src/hsl.rs

```diff
@@ -24,7 +24,7 @@ use crate::{
 /// `Alpha`](struct.Alpha.html#Hsla).
 pub type Hsla<S = Srgb, T = f32> = Alpha<Hsl<S, T>, T>;
 
-/// Linear HSL color space.
+/// HSL color space.
 ///
 /// The HSL color space can be seen as a cylindrical version of
 /// [RGB](rgb/struct.LinRgb.html), where the `hue` is the angle around the color
```

Comment:
```suggestion
/// [RGB](rgb/struct.Rgb.html), where the `hue` is the angle around the color
```
These docs are currently broken in Hsl, Hsv, Hwb.

Reply:
Ah, but they should point to just `Rgb` after these changes, anyway, since none of them are necessarily based on linear RGB anymore. Good catch!

Reply:
Exactly :+1: 

What I also meant was that the current release's docs don't point anywhere  
https://docs.rs/palette/0.5.0/palette/rgb/struct.LinRgb.html

Reply:
Yep, it will be two birds with one stone. Would be nice with a lint for it, but I guess we will just have to wait for intra-rustdoc links first.